src/AutoMapper.Collection.EntityFrameworkCore/AutoMapper.Collection.EntityFrameworkCore.csproj

```
@@ -15,16 +15,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AutoMapper.Collection" Version="5.0.0" />
+    <PackageReference Include="AutoMapper.Collection" Version="7.0.0" />
```

Maybe this should be as below to make the reference only to work with the 7-version of AMC that (if I remember correctly is mapping to specific major version of AM.
```suggestion
    <PackageReference Include="AutoMapper.Collection" Version="[7.0.0,8.0.0)" />
```